### ISSUE TITLE
E-Builder demo

### ISSUE THREAD

**Issue by amenity** [cityofaustin/atd-data-tech#284]
The Corridor Planning Office is has been testing out [E-Builder](https://www.e-builder.net/) and wants to use it for their workflow management. Check out it's project management capabilities to see how they overlap with our MPD needs. 

20 year old end-to-end solution

![Screen Shot 2019-06-27 at 10 46 51 AM](https://user-images.githubusercontent.com/1463708/60308414-3936df00-990e-11e9-804c-2f44b655e42a.png)

**Comment by amenity** [comment]
The functionality demoed was mostly around document workflow management - write, review, comment, change status, etc. 

If we want to look more at their project management capabilities, we should set up another demo.

**Comment by johnclary** [comment]
Agreed that we did not get an understanding of pm capabilities.

We found that development does want it for workflows. Possibly a fairly simple one. 

People doing the demo:
- Jeremy Elliot consultant for CPO
- Jeremiah works for CPO, builds workflows in e-builder

Joe C (Development Review): the need is we want to do development review processes outside of amanada

What e-builder does:
- Structured workflows
- Create audit trail
- Dousing
- No project limit
- Unlimited users

Jeremiah suggests one full-time person supporting ebuilder.

If development review wants to use it, we would do cost sharing with CPO. They just renewed subscription @$125k/yr. Unlimited users.
